contrib/slime-repl.el

```diff
@@ -717,6 +730,7 @@ With prefix argument send the input even if the parenthesis are not
 balanced."
   (interactive "P")
   (slime-check-connected)
+  (setf (mark) slime-repl-input-start-mark)
```

Comment:
What does this bit accomplish?

Reply:
You're absolutely right, this is superfluous.